helm-charts/sep-service/values.yaml

```diff
@@ -0,0 +1,94 @@
+fullName: anchor-platform
+namespace: sandbox
+service:
+   containerPort: 8080
+   servicePort: 8080
```

Comment:
Maybe those don't have to be configurable? Those port numbers are more of an internal implementation detail. I think it might be OK to just use 8080 without even adding option to override in the values file.
But of course if there is a use case for changing ports we should keep those settings